src/java.desktop/windows/classes/sun/awt/windows/WPrinterJob.java

```diff
@@ -995,6 +1009,44 @@ protected void setPolyFillMode(int fillRule) {
         setPolyFillMode(getPrintDC(), fillRule);
     }
 
+    /**
+     * Set the GDI graphics mode to {@code GM_ADVANCED}.
+     */
+    private int setAdvancedGraphicsMode() {
+        return setAdvancedGraphicsMode(getPrintDC());
+    }
+
+    /**
+     * Set the GDI graphics mode.
+     * The {@code mode} should
+     * be one of the following Windows constants:
+     * {@code GM_COMPATIBLE} or {@code GM_ADVANCED}.
+     */
+    private int setGraphicsMode(int mode) {
```

Comment:
Is there any need of "int" return value? I dont see it is used in restoreTransform()

Reply:
I updated the code to return void from setGraphicsMode() method.